services/QuillLMS/client/app/bundles/Evidence/components/studentView/promptStep.tsx

```diff
@@ -53,20 +58,35 @@ export class PromptStep extends React.Component<PromptStepProps, PromptStepState
       customFeedback: null,
       customFeedbackKey: null,
       responseOverCharacterLimit: false,
-      submissionTime: null
+      submissionTime: 0,
+      failedToLoadFeedback: false,
+      currentTime: 0
```

Comment:
I think `currentTime` is a confusing name here because I expect the `currentTime` to be whatever `new Date()).getTime()` returns at a given time (ie, not a stored value). Why didn't it work to just calculate the `currentTime` the way you were before?

Reply:
@emilia-friedberg the component was only updating when the mouse was moving or a keyboard event fired, so if the user just clicks submit and does nothing after, the time wasn't getting calculated. What do you think would be a better variable name? I can update it, just wasn't sure of a better name because at the time of the check, we are checking the current time against submission time